ClosedXML/XLHelper.cs

```diff
@@ -116,8 +116,10 @@ internal static int TrimRowNumber(int rowNumber)
 
         public static bool IsValidColumn(string column)
         {
+            if (String.IsNullOrWhiteSpace(column))
+                return false;
             var length = column.Length;
-            if (String.IsNullOrWhiteSpace(column) || length > 3)
+            if (length > 3)
```

Comment:
Probably not even necessary for a `length` variable?

Reply:
It is used in the code below so I think it's OK to leave it.